server/src/models/errors/dbError.ts

```diff
@@ -1,5 +1,6 @@
-import { LogService } from "./../logic/services/logging.service";
-import { ApiError } from "./api-error";
+import { LogService } from "./../../logic/services/logging.service";
```

Comment:
Yes, that's a good idea. Kebab-case for files and camel-case for classes. Add this changes to another issue.